poi/src/test/java/org/apache/poi/hssf/extractor/TestExcelExtractor.java

```diff
@@ -303,8 +303,8 @@ void testWithEmbeddedInOwn() throws Exception {
 
             // And the base file too
             ex = new ExcelExtractor(fs);
-            assertEquals("Sheet1\nI have lots of embeded files in me\nSheet2\nSheet3\n", ex.getText());
-            assertEquals("Excel With Embeded", ex.getSummaryInformation().getTitle());
+            assertEquals("Sheet1\nI have lots of embeded files in me\nSheet2\nSheet3\n", ex.getText()); /* fixing the spelling here requires editing 44643.xls */
+            assertEquals("Excel With Embeded", ex.getSummaryInformation().getTitle()); /* fixing the spelling here requires editing 44643.xls */
```

Comment:
I'm leaving these here. I'm not sure I have access to the right version of Excel to update the files. But I suspect that the rest of the changes for this commit are ok.